nbs/99_manuscript/lvs/lv136/lv136-traits_assocs.ipynb

```diff
@@ -143,7 +143,7 @@
     {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    phenomexcan_lv_trait_assocs.shape
</pre><p>did you add a bunch more data? why is this so much larger?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/phenoplier/pull/68/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>Great question. The initial implementation of the regression model was very slow. So what we did initially was to run it for each LV-trait pair where the LV was found to be discriminative for that trait cluster, which reduced a lot the number of combinations. Now, the regression model was significantly improved in terms of speed, so we were able to run across all combinations of the 987 LVs and 4,091 trait in PhenomeXcan.</p>